Tentacle/Label.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Label.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-05-23.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+public struct Label: Hashable, CustomStringConvertible {
+    public let URL: NSURL
+    public let name: String
+    public let color: String
```

Comment:
I think we should add a platform-conditionalized `typealias Color = NSColor/UIColor` and make `color` a `Color`.